contrib/ci/generate_docker.py

```
@@ -53,11 +53,18 @@ def get_container_cmd():
 with open(template_file) as file:
     template = file.read()
 
+
+# special case for i386-based debian container
+if TARGET_DISTRO == "debian-i386":
+    template = template.replace("FROM debian:testing", "FROM i386/debian:testing")
+
+
+distro = MATRIX_DISTRO.split("-")[0]
```

Ruff (0.15.11)

[error] Undefined name `MATRIX_DISTRO`

(F821)